docs/LogGuide.md

```diff
@@ -1264,6 +1384,32 @@ You cannot count on the time remaining to be precise.
 In rare cases, the time will already have counted down a tiny bit.
 This matters for cases such as ucob Nael phase doom debuffs.
 
+In some cases, the 'stacks' value may indicate other information about the buff.
+For example, Mudra will show different "stack" values for different combinations of Mudra.
+The only way to ensure that you are getting a "real" stack value is by cross-referencing with game data.
+For example, if you see a stack value of '64',
+but the status effect in question has a maximum stack count of zero,
+then you know it is not a true stack count.
+
+The "Unknown_808" status effect (0x808) uses the 'stacks' field to apply/remove a VFX,
+where the count is the VFX ID.
+
+### Refreshes, Overwrites, and Deaths
+
+If a buff is refreshed early, you will get another 26-line.
+You will not get a 30-line indicating that the existing buff has been removed.
+When stacks of a buff are added or removed, you may or may not receive a removal for the old stack value.
+
+Most debuffs allow one player to place the debuff on each target.
+For some, such as Trick Attack, only one can be on the enemy at a time.
+If a buff is overwritten, a 30-line will be generated for the status effect that got overwritten.
+
+Thus, it is sufficient to track buffs using a combination of caster, target, and status effect ID.
+A refresh or stack change will have the same caster, target, and status effect,
+while an overwrite will generate a 30-line anyway.
+
+When a unit dies, you will get 30-lines for buffs that were removed by it dying.
```

Comment:
`unit` here should probably be either `actor` or `object`, to match with the terminology used in the rest of the guide.

Reply:
Done